be-fe-mono-repo-plain/Jenkinsfile.template

```diff
@@ -1,10 +1,10 @@
 // See https://www.opendevstack.org/ods-documentation/ for usage and customization.
```

Comment:
Maybe we could add in the headers of the templates the explanation you documented (link to it) about the relation of the different version tags. So everyone can know why a specific version number is there.

Reply:
@gerardcl I'm not sure I understand what you want to do ... could you give an example?

Reply:
@michaelsauter 
https://www.opendevstack.org/ods-documentation/opendevstack/3.x/quickstarters/be-golang-plain.html 

so render the tag into it (in this case 3.x) - so people can just copy this line - and are directly in the right doc

Reply:
Hmm, but:

1. over time people update the Jenkinsfile (when they migrate to the next version), so the link gets outdated ... or they also need to maintain the comment ...
2. admins might use e.g. `ODS_IMAGE_TAG=3.0.x`, in which case the generated link would be wrong from the start